backend-go/samples/sub/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"github.com/nats-io/nats.go"
+	"github.com/wuespace/telestion/backend"
+	"log"
+	"os"
+	"os/signal"
+)
+
+const (
+	Subject = "publish_fast"
+)
+
+func main() {
+	log.Println("Setup service")
+	service, err := telestion.StartService()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Println("Subscribing to subject...")
+	counter := 0
+	_, err = service.Nc.Subscribe(Subject, func(msg *nats.Msg) {
+		counter++
+		//log.Printf("Counter: %d", counter)
+		//log.Printf("%s\n", string(msg.Data))
```

Comment:
Let's put these back in. Without them, the sample seemingly does nothing when being run.

Reply:
Fixed in c4cc8e05801472bf157544248dbfd6d77397bc9b